core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/store/dot-container-list.store.ts

```diff
@@ -36,8 +36,11 @@ export class DotContainerListStore extends ComponentStore<DotContainerListState>
         this.route.data
             .pipe(pluck('dotContainerListResolverData'), take(1))
             .subscribe(([isEnterprise, hasEnvironments]: [boolean, boolean]) => {
-                this.updateResolvedProperties([isEnterprise, hasEnvironments]);
-                this.setContainerBulkActions();
+                this.updateResolvedProperties([
```

Comment:
You are still hitting the state twice, in the init and here and you have all to do it once.